Library/Formula/mplayer.rb

```diff
@@ -47,11 +50,15 @@ def install
       --host-cc=#{ENV.cc}
       --disable-cdparanoia
       --prefix=#{prefix}
-      --disable-x11
     ]
 
     args << "--enable-caca" if build.with? "libcaca"
 
+    args << "--enable-x11" if build.with? "x11OSD"
```

Comment:
Done. Thanks for the cluestick hit, the better way worked fine!

On 24 December 2015 at 05:36, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/mplayer.rb
> <https://github.com/Homebrew/homebrew/pull/47287#discussion_r48373477>:
>
> >      ]
> >
> >      args << "--enable-caca" if build.with? "libcaca"
> >
> > +    args << "--enable-x11" if build.with? "x11OSD"
>
> You can stick these in a block...
>
> if build.with? "blah"
>   args << "blah"
>   args << "blah2"
>   args << "blah3"end
>
> And such.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/47287/files#r48373477>.
>